### PR TITLE
Improve tests & success message. Fix total saved size info.

### PR DESCRIPTION
**Pull request by 343dev**

### CODE REVIEW

File: tests/cli.test.js [hunk 1]
```diff
@@ -0,0 +1,607 @@
...
+
+        expectStringContains(stdout, 'Converting 6 images (lossy)...');
+        expectRatio(stdoutRatio, 85, 95);
+        expectFileNotModified(`${fileBasename}.png`);
+        expectFileExists(`${fileBasename}.avif`);
+      });
+    });
+
+    describe('Lossless (--lossless)', () => {
+      // TODO: In lossless mode JPEG file size is always smaller than AVIF
```
Comment (igoradamenko) [inline tests/cli.test.js:190]: @343dev, huh?

File: tests/cli.test.js [hunk 2]
```diff
@@ -0,0 +1,607 @@
...
+
+        expectStringContains(stdout, 'Converting 1 image (lossy)...');
+        expectStringContains(stdout, path.join(tempDir, `${fileBasename}.png`));
+        expectRatio(stdoutRatio, 85, 90);
+        expectFileNotModified(`${fileBasename}.png`);
+        expectFileExists(`${fileBasename}.avif`);
+        expectFileExists(`${fileBasename}.webp`);
+      });
+
+      // TODO: Fix a bug for this test case
```
Comment (igoradamenko) [inline tests/cli.test.js:343]: @343dev, will it be done in a separated PR, or here?

Comment (343dev) [reply tests/cli.test.js]: Fixed it here.

File: tests/cli.test.js (hunk 1 above)
Comment (343dev) [reply tests/cli.test.js:190]: I'll fix it later. I need to do some research.

**Comment by igoradamenko** [comment]
Published @funboxteam/optimizt@3.1.2.

**Pull request merged**